packages/components/src/components/executableDocumentToolbar/executa.ts

```diff
@@ -117,8 +117,10 @@ export const jobToDatum = (job: Job): JobDatum => {
   return DE.toReplete(failedJobGuard(job) ? DE.failure(job) : DE.success(job))
 }
 
+export const jobFetch = wretch().options({ credentials: 'same-origin' })
```

Comment:
As mentioned in another comment, to support cross-origin requests, I believe this needs to be `include` ref https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch